docs/integrators/maiar-integration-react-native-guide.md

```diff
@@ -0,0 +1,344 @@
+# Maiar integration documentation
```

Comment:
I think this documentation will be better placed as `docs/sdk-and-tools/maiar-integration-react-native-guide.md`

Reply:
also, `sidebars.json` has to be updated so your page will be visible in the docs left sidebar

Reply:
Sounds good, will update with the new path.